README.md

```diff
@@ -71,27 +81,61 @@ The UI should be running on
 https://prod.foo.redhat.com:1337/beta/insights/image-builder/landing.
 Note that this requires you to have access to either production or stage (plus VPN and proxy config) of insights.
 
-## Backend Development
+### Backend Development
 
 To develop both the frontend and the backend you can again use the proxy to run both the
 frontend and backend locally against the chrome at cloud.redhat.com. For instructions
 see [devel/README.md](devel/README.md).
 
+## File Structure
+
+### Quick Reference
+| Directory | Description |
+| --------- | ----------- |
+| `/config` | webpack configuration |
+| `/devel` | tools for local development |
+| `/src` | source code |
+| `/src/Components` | source code split by individual components |
+| `/src/test` | test utilities |
+| `/src/test/mocks` | mock handlers and server config for MSW |
+| `/src/store` | Redux store |
+| `/src/api.js` | API calls |
```

Comment:
Maybe add a file link for convenience?
Not sure if directories can be linked to as well, but if so I think that'd be great for user-friendliness.

Reply:
Added the links